.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - "7"
+  - "8.2.1"
```

Comment:
AFAIK Atom is running in node version 6.5.0 and VSCode is running on version 7.4.0, so this change does not make much sense to me. Please revert it and I'll adjust it in the master.